Content.Shared/Backmen/Surgery/Body/SharedBodySystem.Integrity.cs

```diff
@@ -108,33 +125,24 @@ private void ApplyPartDamage(
         bool canSever,
         float partMultiplier)
     {
-        if (
-            partEnt.Comp.Body is not {} body ||
+        if (partEnt.Comp.Body is not {} body ||
             !TryComp<MobStateComponent>(body, out var mobState))
             return;
 
-        foreach (var (damageType, damageValue) in damage.DamageDict)
-        {
-            if (damageValue.Float() == 0
-                || TryEvadeDamage((body, mobState), GetEvadeChance(targetType)))
-                continue;
-
-            var modifier = GetDamageModifier(damageType);
-            var partModifier = GetPartDamageModifier(targetType);
-            var integrityDamage = damageValue.Float() * modifier * partModifier * partMultiplier;
-            TryChangeIntegrity(partEnt,
-                integrityDamage,
-                canSever && _severingDamageTypes.Contains(damageType),
-                targetPart,
-                out var severed);
-
-            if (severed)
-                break;
-        }
+        TryChangeIntegrity(partEnt,
+            damage * partMultiplier,
+            //canSever && damage.DamageDict.Contains<>(_severingDamageTypes),
+            canSever,
+            targetPart,
+            out _);
```

Comment:
I might be dumb, but where are we checking for the Flags on TargetPart to attack one or more parts? Since we did add that to DamageableSystem already.

Reply:
Idk how the flags work tbh 💀

Reply:
Look at how they are used in SharedMoverController. iirc theres some examples there for diagonal walking